packaged/lisk.sh

```diff
@@ -319,12 +324,19 @@ pm2_cleanup() {
 	pm2 kill
 }
 
-check_status() {
-	PM2_PID="$( pm2 jlist |jq -r ".[] | select(.name == \"$PM2_APP\").pm2_env.pm_pid_path" )"
+get_status() {
+	PM2_PID="$( pm2 jlist 2>/dev/null | jq -r ".[] | select(.name == \"$PM2_APP\").pm2_env.pm_pid_path" 2>/dev/null)"
```

Comment:
you can add the flag -s (silent) to pm2 to avoid printing startup messages :)

```
lisk@fra1-fabio-testnet-01:~/lisk-test$ pm2 kill
[PM2] [-] Stopping Modules
[PM2] [v] Modules Stopped
[PM2] [-] Stopping all Applications
[PM2][WARN] No process found
[PM2] [v] All Applications Stopped
[PM2] [-] Stopping Agent
[PM2] [v] Agent Stopped
[PM2] [-] Stopping PM2 Daemon
[PM2] [-] PM2 Daemon Stopped
lisk@fra1-fabio-testnet-01:~/lisk-test$ pm2 jlist|jq
parse error: Invalid numeric literal at line 1, column 5
lisk@fra1-fabio-testnet-01:~/lisk-test$ pm2 kill
[PM2] [-] Stopping Modules
[PM2] [v] Modules Stopped
[PM2] [-] Stopping all Applications
[PM2][WARN] No process found
[PM2] [v] All Applications Stopped
[PM2] [-] Stopping Agent
[PM2] [v] Agent Stopped
[PM2] [-] Stopping PM2 Daemon
[PM2] [-] PM2 Daemon Stopped
lisk@fra1-fabio-testnet-01:~/lisk-test$ pm2 -s jlist|jq
[]
```